cornellh4i/ithaca-recovery
# finished NewMeeting sidebar

### Description
Implemented a responsive template for new meeting sidebar with appropriate padding, margins, spacing per design system.

### Testing
1. Navigate to http://localhost:3000/test
2. Scroll to the bottom of the page, and observe the added section titled "New Meeting Sidebar"

### Notes

- this is only the layout with checkbox acting as our dummy component for now, we will put in actual atoms once they are merged into main
- the height of the sidebar is responsive to the atoms in it and will grow to accommodate the actual atoms once they are merged
![image](https://github.com/user-attachments/assets/b3e75d8a-c470-49d6-8472-73c7e4f21d64)


related to issue #51
